LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 Trevor Brunette
```

Comment:
Can it include others? Maybe something generic enough to include everyone, but not list each out?